spec/features/user_spec.rb

```diff
@@ -63,10 +60,25 @@
   end
 end
 
-describe 'Authentication' do
+describe 'User authentication' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.